gatsby-node.js

```diff
@@ -76,8 +76,8 @@ exports.createPages = ({ boundActionCreators, graphql }) => {
 
     // Create pages for each markdown file.
     posts.forEach(({ node }, index) => {
-      const prev = index === 0 ? false : posts[index - 1].node;
-      const next = index === posts.length - 1 ? false : posts[index + 1].node;
+      const prev = index === 0 ? null : posts[index - 1].node;
```

Comment:
This was causing a type mismatch in the GraphQL schema. I should have used `null` instead of `false`, so in effect, I'm making this node nullable in the GraphQL schema.